pkg/operator/ceph/disruption/clusterdisruption/location.go

```diff
@@ -205,7 +214,7 @@ func getFailureDomainMapForOsds(osdDataList []OsdData, failureDomainType string)
 		}
 	}
 	if len(unfoundOSDs) > 0 {
-		err = fmt.Errorf("failure domain type %s not associated with osds: [%s]", failureDomainType, strings.Join(unfoundOSDs, ","))
+		err = fmt.Errorf("failure domain type %q not associated with osds: %q", failureDomainType, strings.Join(unfoundOSDs, ","))
```

Comment:
nit: whether there is a string slice I think `%v` is best instead of using `strings.Join()`, but either way is fine.